test/unit/middlewares/articlesMiddleware.test.js

```diff
@@ -267,75 +267,13 @@ describe('API endpoint: /api/articles (Middleware test)', () => {
             expect(res).to.have.status(BAD_REQUEST);
             expect(res.body).to.be.an('object');
             expect(res.body).to.haveOwnProperty('code').to.equal(BAD_REQUEST);
-            expect(res.body.message).to.equal('description must be a string');
+            expect(res.body.message).to.equal('The description of this article cannot contain only numbers');
             done();
           });
       });
     });
   });
 
```

Comment:
You deleted 3 valid tests. Why?

Reply:
@thislekan Don't you think '      '   and ' ' are the same? I don't think there is a point repeating it again. I was getting the same response message from both.

Reply:
`'    ' `and` ''` are not the same thing.

Reply:
Once again, great job on the fix. But see the screenshot below for clarity.

<img width="303" alt="Screenshot 2019-04-18 at 7 48 28 PM" src="https://user-images.githubusercontent.com/24186167/56383937-05a34e80-6213-11e9-96bd-fbf47245eb89.png">

Reply:
Okay, @thislekan I have added them back again...